metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/CpuAggregator.java

```diff
@@ -0,0 +1,73 @@
+package io.dockstore.metricsaggregator.helper;
+
+import io.dockstore.metricsaggregator.DoubleStatistics;
+import io.dockstore.openapi.client.model.CpuMetric;
+import io.dockstore.openapi.client.model.Metrics;
+import io.dockstore.openapi.client.model.RunExecution;
+import io.dockstore.openapi.client.model.TaskExecutions;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Aggregate CPU metrics by calculating the minimum, maximum, and average.
+ * @return
+ */
+public class CpuAggregator implements RunExecutionAggregator<CpuMetric, Integer> {
+    @Override
+    public CpuMetric getMetricFromMetrics(Metrics metrics) {
+        return metrics.getCpu();
+    }
+
+    @Override
+    public Integer getMetricFromRunExecution(RunExecution runExecution) {
+        return runExecution.getCpuRequirements();
+    }
+
+    @Override
+    public Optional<RunExecution> getWorkflowExecutionFromTaskExecutions(TaskExecutions taskExecutionsForOneWorkflowRun) {
```

Comment:
This is the new function that aggregates tasks into a workflow execution for the CPU metric